### PR TITLE
fix(card-group): update storybook

### PR DESCRIPTION
**Pull request by ariellalgilmore**
### Related Ticket(s)

#6338 

### Description

Update card-group story to use knobs instead of having different stories.

Screen Shot 2021-12-30 at 8 16 26 AM

Note: 
- cards-per-row knob does not work with narrow grid mode
- same height with tag group with video and images are not aligning

### Changelog

**New**

- prop to card-group `pictograms` so heading for pictogram cards stay aligned to bottom

**Changed**

- knob to choose card types
- knob to add media (alternates images and videos
- min 2 and max 6 to amount of cards
- added `outline` to `grid mode` options, so user can only select one at a time

### CODE REVIEW

**Review by annawen1 — APPROVED** [review]
LGTM!

File: packages/web-components/src/components/card-group/__stories__/card-group.stories.react.tsx [hunk 1]
```diff
@@ -48,184 +47,229 @@ const phraseArray = [
...
+      <DDSCardEyebrow>Topic</DDSCardEyebrow>
+      {tagGroup ? tagGroupContent : ''}
+      <DDSCardCTAFooter cta-type="video" href="1_9h94wo6b" slot="footer"></DDSCardCTAFooter>
     </DDSCardGroupItem>
   );
+
   count = count > 3 ? 0 : count + 1;
-  return defaultCardGroupItem;
+  // eslint-disable-next-line no-nested-ternary
+  return media ? (index % 2 === 0 ? defaultCardGroupItem : videoCardGroupItem) : defaultCardGroupItem;
```
Comment (emyarod) [inline packages/web-components/src/components/card-group/__stories__/card-group.stories.react.tsx:95]:
```suggestion
  return media && index % 2 ? videoCardGroupItem : defaultCardGroupItem;
```

would this be equivalent without nesting ternaries?

**Review by emyarod — COMMENTED** [review]
looks good to me, just wondering would it be worth extracting out the `gridMode` logic to avoid nesting ternaries as well?

**Comment by RichKummer** [comment]
Took a quick look before the last commit: **Card group - Default** looks good, but it looks like there's a missing checkbox for **Card group - with card in card:**
* Missing an _Outlined cards_ checkbox.

Screen Shot 2022-01-04 at 2 26 19 PM

* It also looks like the default Number of cards should be 5, but currently defaults to 3.

**Review by RichKummer — APPROVED** [review]
Looks good! TY for the last change!

**Pull request merged**